src/pages/resultsView/download/DownloadUtils.ts

```diff
@@ -321,6 +322,36 @@ export function generateOtherMolecularProfileDownloadData(
         : [];
 }
 
+export function generateGenericAssayProfileData(
+    molecularProfileId: string[],
+    unfilteredCaseAggregatedData?: CaseAggregatedData<GenericAssayData>
```

Comment:
Are use sure `unfilteredCaseAggregatedData` is optional here?. Looks like its always sent from where its used

Reply:
I see, I copied this from `generateOtherMolecularProfileData`, but seems like for `generic assay`, we don't have a case that will not have `unfilteredCaseAggregatedData`, so it should be required.